libraries/DNSServer/src/DNSServer.cpp

```diff
@@ -60,10 +60,15 @@ DNSServer::DNSServer()
   // They will then report connected.
   _ttl = lwip_htonl(60);
 
+#if !CORE_MOCK
```

Comment:
`rst_info resetInfo{};` in the user_interface.cpp ?

Reply:
https://github.com/esp8266/Arduino/blob/bcb5464167ddf3b9d8035e81e2d5813d98c8461b/cores/esp8266/core_esp8266_main.cpp#L53

It is a snap shot set from `user_init()` in `core_esp8266_main.cpp`

Reply:
Mocking does not build that.